docker/nginx.conf

```diff
@@ -19,102 +21,127 @@ http {
       set $pass http://ui;
       proxy_pass $pass;
       proxy_hide_header Content-Security-Policy;
-      proxy_set_header Host ingress;
+      proxy_set_header Host taskcluster;
+    }
+    location /references {
+      set $pass http://references;
+      proxy_pass $pass;
+      proxy_hide_header Content-Security-Policy;
+      proxy_set_header Host taskcluster;
+    }
+    location /schemas {
```

Comment:
this is used in documentation to show schemas and possibly by some services to pull schemas